tests/main/snap-auto-mount/task.yaml

```diff
@@ -20,7 +23,13 @@ prepare: |
     #shellcheck source=tests/lib/ramdisk.sh
     . "$TESTSLIB/ramdisk.sh"
     setup_ramdisk
-    mkfs.vfat /dev/ram0
+
+    #We use different filesystems to cover both: fat and ext
+    if is_core18_system; then
+        mkfs.ext3 /dev/ram0
```

Comment:
Can we use the same filesystem or is this a part of the test? Where is this coming from?

Reply:
@zyga Yes we can, but I discussed that with @mvo5 and the idea is to use both filesystems (fat is the most popular fs for this and ext3 because fat is not supported on core18 mkfs command).

Reply:
I'll add the explanation in the test